src/PathTracedVolume.ts

```diff
@@ -80,6 +81,7 @@ export default class PathTracedVolume implements VolumeRenderImpl {
   constructor(volume: Volume, settings: VolumeRenderSettings = new VolumeRenderSettings(volume)) {
     this.pathTracingUniforms = pathTracingUniforms();
     this.volume = volume;
+    this.volume.updateRequiredData({ subregion: new Box3(new Vector3(0, 0, 0), new Vector3(1, 1, 1)) });
```

Comment:
Pausing to think about this... creating a PathTracedVolume fires off a load request as a side effect.  I wonder if this should be more explicit (like call it after the ctor finishes and returns) rather than tucked inside here.

Reply:
I did consider adding a method to `VolumeRenderImpl` to contain these side effects, so that it would be possible e.g. to construct an instance without that instance assuming it immediately "owns" the rendering side of its associated `Volume`. I picked "faster" over "cleaner" with this PR, but this seems like a sensible place to go back and make it nice.

Reply:
I was picturing 
```
foo = new PathTracedVolume(volume, ...);
volume.updateRequiredData(...);
```
which hopefully is not a huge difference in code.
I'm not totally 100% which is better but it seems that loosening this coupling might enable more flexibility when it comes to requesting new data from a variety of other kinds of user actions.  If you want to wait till the next phase to reconsider it that makes sense too.

Reply:
My initial concern with that is it makes it less clear which class is responsible for calling `updateRequiredData`. Currently, this method is only ever touched by `VolumeRenderImpl`s, whether during construction or in response to a state update. (The only state update that calls it for now is in `Atlas2DSlice` when the z slice changes, but there may be many more clever optimizations to make using this method in the future.) Moving the method call up splits responsibility for updating required data between `VolumeRenderImpl`s and `VolumeDrawable`.

...though I'm realizing the only further optimizations I can easily think of are awareness of clipping box and visible channels, whose handling is basically identical across `VolumeRenderImpl`s. So maybe it makes more sense than I thought to split responsibilities like that?

Reply:
A few thoughts...

On state updates, I can imagine that multiple dirtyFlags could cause updateRequiredData to be called.  So one change could be to consolidate requests as the dirty flags are checked and then make sure there's nothing redundant or conflicting.  
But also, setTime causes updateRequiredData to be called currently from a much higher level. 

I think it could be much more convenient to be "allowed" to call updateRequiredData from anywhere that has the Volume.  That's assuming the drawable stuff can just "pick up" whatever data there is and draw with it -- which could be one reason to keep the updates held tightly inside the renderimpls. The reason being we don't want to draw with partially old data and partially new.  But of course there are other more explicit ways to do it.  

I generally favor more explicit and separated responsibilities.  Looser coupling, simpler code modules, more predictable behaviors.  I feel like for maintainability it's easier to deal with "do A then do B" as opposed to "do A (and B automatically happened behind the scenes)".

Bottom line, now is the time to try either way.  The code is at a reasonably simple point where it's not a huge cost to switch things around.  So take your pick!

Reply:
I also think that the next round of changes will be more the time to test these assumptions as things go up one level of complexity (caching / prefetching)